src/main/java/org/springframework/integration/aws/inbound/SqsMessageDrivenChannelAdapter.java

```diff
@@ -109,6 +115,7 @@ protected void onInit() {
 		super.onInit();
 		this.listenerContainer = this.simpleMessageListenerContainerFactory.createSimpleMessageListenerContainer();
 		this.listenerContainer.setMessageHandler(new IntegrationQueueMessageHandler());
+		this.listenerContainer.setQueueStopTimeout(this.queueStopTimeout);
```

Comment:
Oh! Sorry, @pfitzsimons-r7 , looks like you have already contributed :smile: .
That's great to see that someone comes back to make more improvement into project.

So, we can't set `queueStopTimeout` unconditionally. By default `long` is `0`, but we can't override that `10000` default. How about to make the field like `Long` and check for `null` here before delegation?
The setter can still be as `long` primitive.

Also, please, take a look during polishing that there are no any other missed properties.